CHANGELOG.md

```diff
@@ -40,15 +39,14 @@ Bug fixes:
 - [libmamba, micromamba] Create empty base prefix with `env update` by @Hind-M in https://github.com/mamba-org/mamba/pull/3519
 - [libmamba, micromamba] fix: Use POSIX-compliant scripts by @jjerphan in https://github.com/mamba-org/mamba/pull/3522
 - [libmamba, micromamba] maint: Clarify `env` subcommand documentation in help menu (cont'd) by @jjerphan in https://github.com/mamba-org/mamba/pull/3539
-- [libmamba] fix: Handle space in `mamba` and `micromamba` executable absolute paths  by @NewUserHa in https://github.com/mamba-org/mamba/pull/3525
+- [libmamba] fix: Handle space in `mamba` and `micromamba` executable absolute paths by @NewUserHa in https://github.com/mamba-org/mamba/pull/3525
```

Comment:
Sometimes prettier fixes double space and some other minor things in markdown text, don't think this is gonna cause trouble